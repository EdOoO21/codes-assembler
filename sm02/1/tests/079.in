-104 -285 220 -667 -507 -497 482 417 12 -683 -865 -282 -940 -590 -775 995 486 5
