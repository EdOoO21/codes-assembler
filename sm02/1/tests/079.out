5
486
995
-775
-590
-940
-282
-865
-683
12
417
482
-497
-507
-667
220
-285
-104
